

drivers/net/wan/farsync.c: In function `fst_intr':
drivers/net/wan/farsync.c:1639: warning: value computed is not used

I assume this is what was intended:

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/wan/farsync.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/net/wan/farsync.c~farsync-warning-fix drivers/net/wan/farsync.c
--- 25/drivers/net/wan/farsync.c~farsync-warning-fix	2004-06-19 15:07:36.390909952 -0700
+++ 25-akpm/drivers/net/wan/farsync.c	2004-06-19 15:08:00.327271072 -0700
@@ -1636,7 +1636,7 @@ fst_intr(int irq, void *dev_id, struct p
 			 */
 			dbg(DBG_TX, "Tx underflow port %d\n", port->index);
                         hdlc_stats(port_to_dev(port))->tx_errors++;
-                        hdlc_stats(port_to_dev(port))->tx_fifo_errors;
+                        hdlc_stats(port_to_dev(port))->tx_fifo_errors++;
 			dbg(DBG_ASS, "Tx underflow on card %d port %d\n",
 			    card->card_no, port->index);
 			break;
_
